pkg/notifications/incorrect_model_status.go

```diff
@@ -0,0 +1,79 @@
+package notifications
+
+import (
+	"context"
+	"log"
+
+	"github.com/google/uuid"
+
+	"github.com/cms-enterprise/mint-app/pkg/graph/model"
+	"github.com/cms-enterprise/mint-app/pkg/models"
+	"github.com/cms-enterprise/mint-app/pkg/sqlutils"
+)
+
+// ActivityIncorrectModelStatusCreate creates an activity for when an incorrect model status is detected.
+// If isLeadFunc returns true for a receiver, a notification is always created regardless of preferences.
+func ActivityIncorrectModelStatusCreate(
+	ctx context.Context,
+	np sqlutils.NamedPreparer,
+	actorID uuid.UUID,
+	receiverIDs []uuid.UUID,
+	phaseSuggestion *model.PhaseSuggestion,
+	modelPlan *models.ModelPlan,
+	getPreferencesFunc GetUserNotificationPreferencesFunc,
+	isLeadFunc IsLeadFunc,
+) (*models.Activity, error) {
+	suggestedStatusStringsRaw := make([]string, len(phaseSuggestion.SuggestedStatuses))
+	suggestedStatusStringsHumanized := make([]string, len(phaseSuggestion.SuggestedStatuses))
+	for i, status := range phaseSuggestion.SuggestedStatuses {
+		suggestedStatusStringsRaw[i] = string(status)
+		suggestedStatusStringsHumanized[i] = status.Humanize()
+	}
+
+	activity := models.NewIncorrectModelStatusActivity(
+		actorID,
+		modelPlan.GetModelPlanID(),
+		string(phaseSuggestion.Phase),
```

Comment:
back to the earlier convo, maybe it makes sense to keep the enum alias instead of raw string? Like you mentioned before, it deserializes ok from the db (it's just a string there)

Reply:
I switched this to be the not humanized strings, unfortunately can't really utilize the phaseSuggesstion model in the model package since the models package (generated types) references the model package